RN v11.4.1 (#422)
mia-platform/documentation: docs/release-notes/v11.4.1.md

---
id: v11.4.1
title: Version v11.4.1 Release Notes
sidebar_label: v11.4.1
image: "img/release-note-link-preview.png"
---

_July 20th, 2023_

## Console

### New Features

#### Added a new event for Console Webhooks: Create Tag

> Via the respective section, it is now possible to configure a webhook on the trigger of a 'Create Tag' event. Thus, by creating a new tag via the design area of the Console, custom webhooks can be triggered if configured appropriately.

Author: ```suggestion
Via the respective section, it is now possible to configure a webhook on the trigger of a 'Create Tag' event. Thus, by creating a new tag via the Design Area of the Console, custom webhooks can be triggered if configured appropriately.
```

### Improvements

#### Merge improvements

* During merge process, users can now accept either all incoming or all current changes instead of resolving diff by diff. More specifically, in each merge modal section, thanks to two proper buttons, it is now possible to decide to either keep current changes (on the right-hand editor) or accept all incoming changes (on the left-hand editor). For more information about how to perform a merge of configurations, [go to the related documentation](/development_suite/api-console/api-design/merge_collaboration.md).

* In merge modal, Authorization section has been organized into four different subsections: Enabled Services, Image Name, Permissions, RBAC storage. In particular, within Permissions section, users can now view and manage diffs for Policies and Policy Tests.

### Bug Fix

> This version addressed the following bugs:

Author: ```suggestion
This version addressed a bug, listed here below:
```

The button for generating a back-kit kit data schema from a CRUD collection is now always visible, albeit disabled if
some condition is not meet. Moreover, the generation algorithm has been improved to better handle CRUD "base" properties
and properties of type `ObjectId` and `Geopoint`.

#### Bug Fix

##### Advanced tab "edited" state is now displayed correctly

The blue dot in the Advanced tabs showing that there are some pending modifications no longer appears when working in
different sections of the Configurator.

##### Properties of type dynamic-icon are now properly configurable

A bug preventing icons lists to be searched when configuring properties of type `dynamic-icon` has been solved. Moreover,
a preview of the available icons is now correctly shown before the icon name.

### Backoffice Library - 1.3.17

#### Improvements

##### `bk-layout` supports `phosphor` icons

`bk-layout` supports `phosphor` icons in menu items via `@micro-lc/iconic@1.3.0`

#### Bug Fix

##### `bk-layout` icon style

`bk-layout` icons style look and feel is now using the primary color

## Marketplace

### Marketplace Updates

#### Swagger aggregator - v3.5.0

> - Added `PREVENT_REQUEST_BODY_CONFLICTS` env variable. When this boolean is set to true, it disables the duplication of the `requestBodies`object when converting from swagger to openApi V2. This avoids conflict between multiple requests with the same name inside the `components` object.

Author: ```suggestion
Added `PREVENT_REQUEST_BODY_CONFLICTS` env variable. When this boolean is set to true, it disables the duplication of the `requestBodies`object when converting from swagger to openApi V2. This avoids conflict between multiple requests with the same name inside the `components` object.
```

---
id: v11.4.1
title: Version v11.4.1 Release Notes
sidebar_label: v11.4.1
image: "img/release-note-link-preview.png"
---

_July 20th, 2023_

## Console

### New Features

#### Added a new event for Console Webhooks: Create Tag

Via the respective section, it is now possible to configure a webhook on the trigger of a 'Create Tag' event. Thus, by creating a new tag via the Design Area of the Console, custom webhooks can be triggered if configured appropriately.

> For further details, [go to the documentation](/development_suite/webhooks-and-events/events.md).

Author: ```suggestion
For further details, [go to the documentation](/development_suite/webhooks-and-events/events.md#tag-created).
```

> For further details, [go to the documentation](/development_suite/webhooks-and-events/events.md#tag-created).

Author: ```suggestion
For further details, [go to the documentation](/development_suite/webhooks-and-events/events.mdx#tag-created).
```

---
id: v11.4.1
title: Version v11.4.1 Release Notes
sidebar_label: v11.4.1
image: "img/release-note-link-preview.png"
---

_July 20th, 2023_

## Console

### New Features

> #### Create Tag as new event for Console Webhooks

Reviewer: ```suggestion
#### Added a new event for Console Webhooks: Create Tag
```

> Via the respective section, it is now possible to configure a webhook on the trigger of a 'Create Tag' event. Thus, by creating a new tag via the Console, custom webhooks can be triggered if configured appropriately.

Reviewer: ```suggestion
Via the respective section, it is now possible to configure a webhook on the trigger of a 'Create Tag' event. Thus, by creating a new tag via the design area of the Console, custom webhooks can be triggered if configured appropriately.
```

### Improvements

#### Merge improvements

* During merge process, users can now accept either all incoming or all current changes instead of resolving diff by diff. More specifically, in each merge modal section, thanks to two proper buttons, it is now possible to decide to either keep current changes (on the right-hand editor) or accept all incoming changes (on the left-hand editor).

* In merge modal, Authorization section has been organized into four different subsections: Enabled Services, Image Name, Permissions, RBAC storage. In particular, within Permissions section, users can now view and manage diffs for Policies and Policy Tests.

### Bug Fix

This version addressed the following bugs:

* It has been fixed a bug that prevented the creation and deletion of root service accounts

> 
> * 

Reviewer: ```suggestion
```

## Fast Data

> 
> ### New Features
> 
> ### Improvements

Reviewer: ```suggestion
```

library.

##### Automatic data schema generation from CRUD has been improved

The button for generating a back-kit kit data schema from a CRUD collection is now always visible, albeit disabled if
some condition is not meet. Moreover, the generation algorithm has been improved to better handle CRUD "base" properties
and properties of type `ObjectId` and `Geopoint`.

#### Bug Fix

##### Advanced tab "edited" state is now displayed correctly

The blue dot in the Advanced tabs showing that there are some pending modifications no longer appears when working in
different sections of the Configurator.

##### Properties of type dynamic-icon are now properly configurable

A bug preventing icons lists to be searched when configuring properties of type `dynamic-icon` has been solved. Moreover,
a preview of the available icons is now correctly shown before the icon name.

### Backoffice Library - 1.3.17

#### Improvements

##### `bk-layout` supports `phosphor` icons

`bk-layout` supports `phosphor` icons in menu items via `@micro-lc/iconic@1.3.0`

#### Bug Fix

##### `bk-layout` icon style

`bk-layout` icons style look and feel is now using the primary color

> 
> ## Marketplace
> 
> ### Marketplace Updates

Reviewer: ```suggestion

## Marketplace

### Marketplace Updates
```

## How to update your Console

> For on-premise installations, please contact your Mia-Platform referent and upgrade your _Console Helm Chart_ to `v9.X.X`.

Reviewer: ```suggestion
For on-premise installations, please contact your Mia-Platform referent and upgrade your _Console Helm Chart_ to `v9.5.0`.
```
